### DISCUSSION TITLE
Intallation on Mac

### DISCUSSION THREAD

**Discussion by v1cgonz** [NOAA-EMC/WW3#1084] [Q&A]
Hello everyone, I'm learning how to use the model and I'm having problems with the installing, I have MAC, I followed all the steps I saw and nothing works, as I read, after having installed it, I should have the ww3_grid file, but I haven't. Does anyone know how to install this model on mac? Thanks for the answers

**Comment by aliabdolali** [comment]
If you have Hebrew, you can follow the steps described here https://github.com/erdc/WW3/wiki/FAQs%E2%80%90page#for-macosx-with-gfortran and compile it. I tested it on different Macs w/o a problem

**Reply by v1cgonz** [reply]
Thank you so much, I'm now trying and still getting errors
Is a little confused for me that section

I don't understand what I supossed to do here: 
switch<user_defined> switch

What does this part do? what is "switch"?

**Reply by aliabdolali** [reply]
You need to define the path to the switch file, which contains the options you want to use for your simulation for physics, numerical schemes, ...
There are plenty of examples under model/bin and regtests/*/input. You need to use the appropriate one for your case.
